go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/openshift/api v3.9.1-0.20191111211345-a27ff30ebf09+incompatible
 	github.com/openshift/library-go v0.0.0-20210727084322-8a96c0a97c06
 	github.com/openshift/ocs-operator v0.0.1-alpha2
-	github.com/ramendr/ramen v0.0.0-20220114120954-238a9a87a098
+	github.com/ramendr/ramen v0.0.0-20220201125512-ff29bf0d3dbb
```

Comment:
From https://github.com/RamenDR/ramen/commit/6ae35d632637dae22a9924d7aa2121b867e12544 (Feb3), ramen did some major version changes on its dependency. Things are started breaking on ODFMO side with the latest ramen master. 

I have taken a commit from https://github.com/RamenDR/ramen/commit/ff29bf0d3dbbcd116dc683c3e91d7152e22e8d3b (Feb), which is a previous commit from the Feb3 commit.

It has all fields from ramenConfig, I cross verified the entire file.

Reply:
But if ramen did any changes on ramenConfig file then we  need to resolve this  dependency problem

Reply:
@umangachapagain You suggestion please

Reply:
Since we are using only the API, let's pull only that package.

Reply:
The key dep to change is possibly `github.com/openshift/hive/apis => github.com/openshift/hive/apis v0.0.0-20210415080537-ea6f0a2dd76c` in `go.mod`. BUT, that can be a separate PR. The update to stolostron repo is required to keep deps in ramen even with RHACM 2.4